tests/phpunit/tests/Story_Query.php

```diff
@@ -78,7 +78,9 @@ public static function wpSetUpBeforeClass( $factory ) {
 			'show_title'                => false,
 			'show_author'               => false,
 			'show_date'                 => false,
+			'show_excerpt'              => false,
 			'show_stories_archive_link' => false,
+			'has_square_corners'        => false,
```

Comment:
What is the `has_square_corners` option used for / where is it used?

Reply:
This option defines whether to show curved corners or sharp one for the stories. Every 'renderer' instance is created and passed this `Story_Query` object. so you can consider this as defaults that will get passed to the `Renderer`.
And in turn 'Renderer' add a class if it is set to true.

Reply:
In which Renderer is `has_square_corners` `true? Just so I can take a look.

Is this an actual option in the UI?

Reply:
Check the block inspector controls 'Show sharp corners', here
> https://google-webstories.rt.gw/wp-admin/post.php?post=1099&action=edit
This toggle option is visible for all view types except `circles`.